spec/daffy_lib/caching_encryptor_spec.rb

```diff
@@ -135,6 +137,7 @@
 
   describe '#decrypt' do
     decryption_params.each do |param|
+      # rubocop:disable RSpec/RepeatedDescription
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary disabling of `RSpec/RepeatedDescription`.](https://app.codacy.com/gh/Zetatango/daffy_lib/pullRequest?prid=4990115)